lib/angular_ast.dart

```diff
@@ -16,30 +16,41 @@ export 'package:angular_ast/src/ast.dart'
         EventAst,
         ExpressionAst,
         InterpolationAst,
+        ParsedAttributeAst,
+        ParsedBananaAst,
+        ParsedEventAst,
+        ParsedElementAst,
+        ParsedPropertyAst,
+        ParsedReferenceAst,
+        ParsedStarAst,
         PropertyAst,
         ReferenceAst,
         StandaloneTemplateAst,
         StarAst,
         SyntheticTemplateAst,
         TemplateAst,
-        TextAst;
+        TextAst,
+        WhitespaceAst;
 export 'package:angular_ast/src/lexer.dart' show NgLexer;
 export 'package:angular_ast/src/parser.dart' show NgParser;
-export 'package:angular_ast/src/token.dart' show NgToken, NgTokenType;
+export 'package:angular_ast/src/token/tokens.dart'
+    show NgToken, NgTokenType, NgAttributeValueToken, NgSpecialAttributeToken;
 export 'package:angular_ast/src/visitor.dart'
     show
         HumanizingTemplateAstVisitor,
         IdentityTemplateAstVisitor,
-        TemplateAstVisitor;
+        TemplateAstVisitor,
+        DesugarVisitor;
 
 /// Returns [template] parsed as an abstract syntax tree.
-List<TemplateAst> parse(
-  String template, {
-  @required String sourceUrl,
-  bool toolFriendlyAst: false,
-}) {
+List<TemplateAst> parse(String template,
+    {@required String sourceUrl,
+    bool toolFriendlyAst: false,
+    bool desugar: false}) {
```

Comment:
Can you document in `parse` what `desugar` means.

We might also want to default this to `true`, as I imagine most end users want to consume, not analyze, the AST.